eng/pipelines/runtimelab/install-nodejs.ps1

```diff
@@ -33,9 +47,17 @@ if ($RetryCount -le 0)
     exit 1
 }
 
-Expand-Archive -LiteralPath "$InstallPath\$NodeJSInstallName.zip" -DestinationPath $InstallPath -Force
+if ($IsWindows)
+{
+    Expand-Archive -LiteralPath "$InstallPath\$NodeJSZipName" -DestinationPath $InstallPath -Force
+    $NodeJSExePath = "$InstallPath\$NodeJSInstallName\node.exe"
```

Comment:
Does `Expand-Archive` not work on Linux (it's a bit surprising)?

Reply:
It does, but doesn't support `gz` which is gzip, or `xz` which is some LZMA I think.

Reply:
And those are out options for linux https://nodejs.org/dist/v9.9.0/